docs/sql-ref-ansi-compliance.md

```diff
@@ -9,9 +9,9 @@ license: |
   The ASF licenses this file to You under the Apache License, Version 2.0
   (the "License"); you may not use this file except in compliance with
   the License.  You may obtain a copy of the License at
- 
```

Comment:
The changes are unnecessary, this might just cause conflicts. You can open a separate PR and fix the issue in all *.md.

Reply:
accidentaly pushed this, my bad, will revert